forgotPassword.js

```diff
@@ -1,39 +1,35 @@
-document.getElementById("send_code_button_forgot_password_page").addEventListener("click", onSendCodeClick);
-document.getElementById("log_in_confirmation_button_log_in_page").addEventListener("click", onCodeConfirmClick);
+document.getElementById("send_code_btn_fp_pg").addEventListener("click", onSendCodeClick);
+
 function onSendCodeClick() {
-    const user_email = document.getElementById("input_forgot_password_page").value;
-    const span_user_email = document.getElementById("user_email_forgot_password_page");
+
+    const user_email = document.getElementById("em_input_fp_pg").value;
+    const span_user_email = document.querySelector("#confirm_txt span");
+    document.getElementById("confirm_txt").style.display = "inline";
+    document.getElementById("confirm_txt").classList.add("animate__animated");
+    document.getElementById("confirm_txt").classList.add("animate__fadeInLeft");
     span_user_email.textContent = user_email;
-    document.getElementById("confirmation_text").classList.add("inline_class");
-    document.getElementById("input_email").innerHTML = `
-            <div class="row justify-content-center">
-                <div class="d-flex flex-wrap flex-column align-items-center col-xxl-4 col-xl-5 col-lg-6 col-md-6 col-sm-10 col-10"
-                     id="background_input_forgot_password_page">
-                    <md-outlined-text-field class="col-12" id="input_forgot_password_page" label="Confirmation code"></md-outlined-text-field>
-                </div>
-            </div>
-             <div class="row justify-content-center">
-                <md-filled-button id="log_in_confirmation_button_log_in_page"
-                                  class="col-xxl-2 col-xl-2 col-lg-2 col-md-3 col-sm-8 col-8"
-                                  label="Confirm"></md-filled-button>
-            </div>
-            `;
+    const em_input_form = document.getElementById("input_email");
+    while (em_input_form.firstChild) {
+        em_input_form.removeChild(em_input_form.firstChild);
+    }
+    em_input_form.insertAdjacentHTML("afterbegin", "            <div class=\"row justify-content-center animate__animated animate__fadeInLeft\">\n" +
```

Comment:
Could we have this code in html? But hide it when it is not need and show when it need? It may help you: css property `display: none`

Reply:
Done